sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/SchemaRegistryAsyncClient.java

```diff
@@ -338,7 +432,30 @@ private static Throwable remapError(ErrorException error) {
         return error;
     }
 
-    private static String getContentType(SchemaFormat schemaFormat) {
-        return "application/json; serialization=" + schemaFormat;
+    /**
+     * Converts an input stream into its string representation.
+     *
+     * @param inputStream Input stream.
+     *
+     * @return A string representation.
+     *
+     * @throws UncheckedIOException if an {@link IOException} is thrown when creating the readers.
+     */
+    private static String convertToString(InputStream inputStream) {
+        final StringBuilder builder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(
+            new InputStreamReader(inputStream, StandardCharsets.UTF_8))) {
+
+            String str;
+
+            while ((str = reader.readLine()) != null) {
+                builder.append(str);
+            }
+
+        } catch (IOException exception) {
+            throw new UncheckedIOException("Error occurred while deserializing schemaContent.", exception);
+        }
+
```

Comment:
Is it possible to use `readAllBytes()` API?

```java
        try {
            return new String(inputStream.readAllBytes(), StandardCharsets.UTF_8);
        } catch (IOException exception) {
            throw new UncheckedIOException("Error occurred while deserializing schemaContent.", exception);
        }
```

Reply:
Ah yeah. This was introduced in JDK 9, so I didn't use it.

Reply:
I see, thanks!